src/Microsoft.Health.SqlServer/Features/Schema/SchemaJobWorker.cs

```diff
@@ -70,11 +70,13 @@ public async Task ExecuteAsync(SchemaInformation schemaInformation, string insta
                         int? previous = schemaInformation.Current;
                         schemaInformation.Current = await schemaDataStore.UpsertInstanceSchemaInformationAsync(instanceName, schemaInformation, cancellationToken);
 
-                        // If there was a change in the schema version
-                        if (previous != schemaInformation.Current)
+                        // If there was a change in the schema version and this isn't the base schema
+                        if (schemaInformation.Current != previous && schemaInformation.Current > 0)
                         {
+                            var isFullSchemaSnapshot = previous == null;
+
                             // Fire a notification
```

Comment:
nit: probably we can remove this comment as _mediator.NotifySchemaUpgradedAsync states the purpose itself.

Reply:
Good point, thanks. Fixed in a70b4eb.